media/js/src/simulationOne/criticalValue.jsx

```diff
@@ -67,6 +67,20 @@ export const CriticalValue = ({
         setCompareCritical(newValue);
     };
 
+    const handleRedo = () => {
```

Comment:
Because this is a handler for clicking an `a` tag, it may be best to prevent the default click behavior for it:

```
const handleRedo = (e) => {
    e.preventDefault();
    setIsRedo(true);
};

Reply:
Ah. Thanks Nik!